Makefile.gh

```diff
@@ -23,7 +23,7 @@ codespell:
 
 bandit:
 	pip install bandit
-	bandit -o bandit-output.txt -r --skip B101,B105,B311,B404,B603 .
+	bandit -r --skip B101,B105,B311,B404,B603 .
```

Comment:
Hi @clebergnu 
Right now, `bandit-output.txt` is stored as an artifact so the developer can download the file and check it locally.
So ideally this file is still saved even when the results are shown to the screen.

Reply:
See https://github.com/avocado-framework/avocado/blob/master/.github/workflows/ci.yml#L224